test/E2ETest/Microsoft.Test.E2E.AspNet.OData/QueryComposition/FilterTests.cs

```diff
@@ -421,6 +442,32 @@ protected override void UpdateConfiguration(WebRouteConfiguration configuration)
             }
         }
 
+        [Fact]
+        public async Task TestInOperator()
+        {
+            // While this seems ideal for a Theory test case, the IEnumerable<Movie> would need to be serialize-able in
+            // order to generate an Xunit 2.0 test case.
+            var testCases = InOperatorData;
```

Comment:
That's strange in this file.

Can not we use [Theory]?
Would you please help check?

Reply:
Thanks for the comment.

The data used for expected results is dynamically generated so [Theory] would not work based on the [xunit docs](https://andrewlock.net/creating-parameterised-tests-in-xunit-with-inlinedata-classdata-and-memberdata/) I've read. Therefore we would have to use [ClassData], and that is essentially what we are currently doing so I will leave this as-is.